rgdc/rgdc/utils.py

```diff
@@ -122,3 +127,53 @@ def download_checksum_file_to_path(file: Dict, path: Path, keep_existing: bool):
 def spatial_subentry_id(search_result):
     """Get the id of a returned SpatialEntry."""
     return search_result['subentry_pk']
+
+
+def spatial_search_params(
+    self,
```

Comment:
One last thing, this parameter needs to be removed
```suggestion
```

Reply:
Thanks!